test/docker/base/run_test_wrapper.sh

```diff
@@ -24,11 +24,13 @@ cd /galaxy
 GALAXY_CONFIG_OVERRIDE_DATABASE_CONNECTION="$GALAXY_TEST_DBURI";
 export GALAXY_CONFIG_OVERRIDE_DATABASE_CONNECTION
 
+[ ! -d .venv ] && ./scripts/common_startup.sh
```

Comment:
This is probably wrong since it will cause `common_startup.sh` to never be run again after `.venv` is created. Of course, fixing it means building another image...